### PR TITLE
[MIG][14.0] portal_odoo_debranding

### PR DESCRIPTION
**Pull request by ivantodorovich**
Standard migration

### CODE REVIEW

**Review by hparfr — APPROVED** [review]

**Comment by francesco-ooops** [comment]
any chance to merge this?

**Comment by meigallodixital** [comment]
Will merge soon this?

**Comment by ivantodorovich** [comment]
It needs one more review (2 required)

Feel free to review it yourself ☺️ anyone can do it.

If you don't have technical knowledge, you can do a functional test on runbot. These tests are often more valuable than code reviews

**Comment by pedrobaeza** [comment]
https://odoo-community.org/page/review

**Review by francesco-ooops — APPROVED** [review]
LGTM

File: portal_odoo_debranding/i18n/portal_odoo_debranding.pot
Comment (Shide) [inline portal_odoo_debranding/i18n/portal_odoo_debranding.pot:6]: Change 12 to 14

File: portal_odoo_debranding/README.rst
Comment (Shide) [inline portal_odoo_debranding/README.rst:17]: Change 12 to 14

File: portal_odoo_debranding/README.rst
Comment (Shide) [inline portal_odoo_debranding/README.rst:20]: Change 12 to 14

File: portal_odoo_debranding/README.rst
Comment (Shide) [inline portal_odoo_debranding/README.rst:23]: Change 12 to 14

File: portal_odoo_debranding/README.rst
Comment (Shide) [inline portal_odoo_debranding/README.rst:41]: Change 12 to 14

File: portal_odoo_debranding/README.rst
Comment (Shide) [inline portal_odoo_debranding/README.rst:81]: Change 12 to 14

File: portal_odoo_debranding/static/description/index.html
Comment (Shide) [inline portal_odoo_debranding/static/description/index.html:389]: Change 12 to 14

File: portal_odoo_debranding/static/description/index.html
Comment (Shide) [inline portal_odoo_debranding/static/description/index.html:418]: Change 12 to 14

File: portal_odoo_debranding/__manifest__.py
Comment (Shide) [inline portal_odoo_debranding/__manifest__.py:6]: You can remove development_status beta if you want (not blocking this)

**Review by Shide — CHANGES_REQUESTED** [review]
Wrong type of review.

**Review by Shide — APPROVED** [review]
Seems good to me

**Comment by yajo** [comment]
/ocabot merge nobump

Let's try after merging #46

**Pull request merged**
